scripts/jenkins/ardana/ansible/roles/ardana_deploy/files/enable-centos-rpms-on-rhel/roles/deployer-rhel-repo/vars/main.yml

```diff
@@ -88,7 +88,7 @@ rhel_repo_rpms_list:
   - python-keystoneauth1
   - python-keystoneclient
   - python-monotonic
-  - python-msgpack-  - python
+  - python-msgpack-python
```

Comment:
for rocky this should actually be python-msgpack, not python-msgpack-python (old deprecated package which was renamed to python-msgpack)